docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/install-vertex.md

```diff
@@ -235,7 +281,35 @@ class MessageBrokerDependencyProvider extends SprykerMessageBrokerDependencyProv
 
 ```
 
-### 5. Optional: To send invoices to Vertex through OMS, configure your Payment OMS
+### 5. Configure the OMS dependency provider
+
+Add the following code to `src/Pyz/Zed/Oms/OmsDependencyProvider.php`:
```

Comment:
BTW in the next section we already have these words

>Configure the Oms dependency provider
>Add the config to src/Pyz/Zed/Oms/OmsDependencyProvider.php:

Reply:
This section is marked as optional and I'm not sure we treat refunds as optional feature.

Reply:
Refunds are optional, moved the config section